libs/shared/data-access/work-api/src/lib/models/form/form.model.ts

```diff
@@ -22,7 +35,12 @@ export type IActiveForm = Record<
 export class FormModel implements SchemaModel<IForm> {
   public formConfigurationModel: FormConfigurationModel = new FormConfigurationModel();
   public taskName = '';
-
+  public name?: string | null = '';
```

Comment:
These should all be like this, empty string should be there empty value
```suggestion
  public name = '';
```